_data/security.yml

```
@@ -158,45 +158,45 @@ websites:
     tfa:
       - sms
       - phone
-      - software
+      - proprietary
       - hardware
```

Let's fix the hardware tag as well.
```suggestion
      - u2f
```